include/shmem.h.in.tpl

```diff
@@ -102,10 +102,10 @@ typedef struct {
 #define SHMEM_CTX_SERIALIZED 0x001001L
 #define SHMEM_CTX_PRIVATE 0x001002L
 #define SHMEM_CTX_NOSTORE 0x001003L
-#define SHMEM_CTX_INVALID (-1L)
 
 typedef void* shmem_ctx_t;
 #define SHMEM_CTX_DEFAULT (shmem_ctx_t) 0x80000
+#define SHMEM_CTX_INVALID NULL
```

Comment:
I'm going to add a new commit with this fix. I chose the value of `NULL` for symmetry with `SHMEM_TEAM_INVALID` (and not because of my advocacy for using NULL for invalid/null handles).

The reason I moved this down is because `SHMEM_CTX_INVALID` simply belongs together with `SHMEM_CTX_DEFAULT`, both are values for `shmem_ctx_t` handles, cf the declaration of the `SHMEM_TEAM_XXX` handles.